chart/templates/_helpers.tpl

```diff
@@ -134,10 +134,14 @@ Generate the secret def to be used in pods definitions
 {{- end -}}
 
 {{/*
-Create gcpBucketsProjectId using the gcpBucketsProjectId config or, if not defined, selfHostedGcpProjectId.
+Return true if a `appSecrets.gcpBucketsServiceAccountKey` is specified in any way
 */}}
-{{- define "carto.gcpBucketsProjectId" -}}
-{{ default .Values.cartoConfigValues.selfHostedGcpProjectId .Values.appConfigValues.gcpBucketsProjectId }}
+{{- define "carto.gcpBucketsServiceAccountKey.used" -}}
+{{- if .Values.appSecrets.gcpBucketsServiceAccountKey.existingSecret.name }}
```

Comment:
Use an OR?

```suggestion
{{- if or .Values.appSecrets.gcpBucketsServiceAccountKey.existingSecret.name .Values.appSecrets.gcpBucketsServiceAccountKey.existingSecret.name }}
true
```

Reply:
I tried it, but since they are non-boolean values it can't be used like this and it fails  T.T

Reply:
```
{{- if or (eq .Values.isCar true) (eq .Values.isBus true) }}
```